### DISCUSSION TITLE
Email and phone number in the receipt

### DISCUSSION THREAD

**Discussion by reissgo** [Glench/ExtPay#168] [Q&A] [answered]
I just did a test purchase through ExtensionPay and shortly after received a receipt 
from Stripe containing the the following: 

     If you have any questions, contact us at EMAIL_ADDRESS or call at PHONE_NUMBER.

My question is, can I change the EMAIL_ADDRESS and/or PHONE_NUMBER in future receipts?
I don't know if they were set by ExtensionPay or Stripe.

**Comment by reissgo** [comment, marked as answer]
...after much thrashing around I think I found the answer. On the Stripe dashboard I click the settings icon then click "public details" this gives you a page where you can edit the "support email" and the "support phone number" which can be switched off altogether for receipts and invoices.

**Reply by Glench** [reply]
Glad you found your answer!
